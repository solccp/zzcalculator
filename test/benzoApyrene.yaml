---
structure_id: 1
number_of_atoms: 20
double_bonds:
- [1, 6]
- [2, 3]
- [10, 9]
- [20, 19]
- [18, 12]
- [17, 13]
- [14, 8]
- [7, 11]
- [16, 15]
- [4, 5]
---
structure_id: 2
number_of_atoms: 20
double_bonds:
- [1, 6]
- [2, 3]
- [10, 9]
- [20, 19]
- [18, 12]
- [17, 13]
- [14, 8]
- [7, 4]
- [11, 16]
- [15, 5]
---
structure_id: 3
number_of_atoms: 20
double_bonds:
- [1, 6]
- [2, 3]
- [10, 9]
- [20, 19]
- [18, 12]
- [17, 13]
- [14, 8]
rings:
- [7, 4, 5, 15, 16, 11]
---
structure_id: 4
number_of_atoms: 20
double_bonds:
- [1, 6]
- [2, 3]
- [10, 9]
- [20, 18]
- [19, 17]
- [13, 14]
- [12, 11]
- [16, 15]
- [8, 7]
- [5, 4]
---
structure_id: 5
number_of_atoms: 20
double_bonds:
- [1, 6]
- [2, 3]
- [10, 9]
- [20, 18]
- [19, 17]
- [13, 12]
- [14, 8]
- [4, 7]
- [5, 15]
- [11, 16]
---
structure_id: 6
number_of_atoms: 20
double_bonds:
- [1, 6]
- [2, 3]
- [10, 9]
- [20, 18]
- [19, 17]
- [13, 12]
- [14, 8]
- [4, 5]
- [7, 11]
- [16, 15]
---
structure_id: 7
number_of_atoms: 20
double_bonds:
- [1, 6]
- [2, 3]
- [10, 9]
- [20, 18]
- [19, 17]
- [13, 12]
- [14, 8]
rings:
- [4, 5, 15, 16, 11, 7]
---
structure_id: 8
number_of_atoms: 20
double_bonds:
- [1, 6]
- [2, 3]
- [10, 9]
- [20, 18]
- [19, 17]
- [4, 5]
- [16, 15]
rings:
- [13, 12, 11, 7, 8, 14]
---
structure_id: 9
number_of_atoms: 20
double_bonds:
- [1, 6]
- [2, 3]
- [10, 9]
- [14, 8]
- [4, 7]
- [5, 15]
- [16, 11]
rings:
- [20, 18, 12, 13, 17, 19]
---
structure_id: 10
number_of_atoms: 20
double_bonds:
- [1, 6]
- [2, 3]
- [10, 9]
- [14, 8]
- [4, 5]
- [7, 11]
- [16, 15]
rings:
- [20, 18, 12, 13, 17, 19]
---
structure_id: 11
number_of_atoms: 20
double_bonds:
- [1, 6]
- [2, 3]
- [10, 9]
- [14, 8]
rings:
- [20, 18, 12, 13, 17, 19]
- [4, 5, 15, 16, 11, 7]
---
structure_id: 12
number_of_atoms: 20
double_bonds:
- [1, 2]
- [6, 5]
- [15, 16]
- [3, 10]
- [4, 7]
- [9, 8]
- [14, 13]
- [17, 19]
- [11, 12]
- [20, 18]
---
structure_id: 13
number_of_atoms: 20
double_bonds:
- [1, 2]
- [6, 5]
- [15, 16]
- [3, 4]
- [10, 9]
- [19, 20]
- [18, 12]
- [11, 7]
- [17, 13]
- [14, 8]
---
structure_id: 14
number_of_atoms: 20
double_bonds:
- [1, 2]
- [6, 5]
- [15, 16]
- [3, 4]
- [10, 9]
- [19, 17]
- [20, 18]
- [14, 13]
- [12, 11]
- [8, 7]
---
structure_id: 15
number_of_atoms: 20
double_bonds:
- [1, 2]
- [6, 5]
- [15, 16]
- [3, 4]
- [10, 9]
- [19, 17]
- [20, 18]
- [14, 8]
- [7, 11]
- [12, 13]
---
structure_id: 16
number_of_atoms: 20
double_bonds:
- [1, 2]
- [6, 5]
- [15, 16]
- [3, 4]
- [10, 9]
- [19, 17]
- [20, 18]
rings:
- [14, 8, 7, 11, 12, 13]
---
structure_id: 17
number_of_atoms: 20
double_bonds:
- [1, 2]
- [6, 5]
- [15, 16]
- [3, 4]
- [10, 9]
- [11, 7]
- [14, 8]
rings:
- [19, 17, 13, 12, 18, 20]
---
structure_id: 18
number_of_atoms: 20
double_bonds:
- [1, 2]
- [6, 5]
- [15, 16]
- [11, 12]
- [18, 20]
- [19, 17]
- [14, 13]
rings:
- [3, 4, 7, 8, 9, 10]
---
structure_id: 19
number_of_atoms: 20
double_bonds:
- [10, 9]
- [15, 16]
- [7, 11]
- [8, 14]
- [17, 19]
- [13, 12]
- [18, 20]
rings:
- [1, 2, 3, 4, 5, 6]
---
structure_id: 20
number_of_atoms: 20
double_bonds:
- [10, 9]
- [15, 16]
- [7, 11]
- [8, 14]
- [17, 13]
- [19, 20]
- [12, 18]
rings:
- [1, 2, 3, 4, 5, 6]
---
structure_id: 21
number_of_atoms: 20
double_bonds:
- [10, 9]
- [15, 16]
- [7, 11]
- [8, 14]
rings:
- [1, 2, 3, 4, 5, 6]
- [17, 13, 12, 18, 20, 19]
---
structure_id: 22
number_of_atoms: 20
double_bonds:
- [10, 9]
- [15, 16]
- [7, 8]
- [11, 12]
- [18, 20]
- [19, 17]
- [14, 13]
rings:
- [1, 2, 3, 4, 5, 6]
---
structure_id: 23
number_of_atoms: 20
double_bonds:
- [10, 9]
- [15, 16]
- [18, 20]
- [19, 17]
rings:
- [1, 2, 3, 4, 5, 6]
- [7, 8, 14, 13, 12, 11]
