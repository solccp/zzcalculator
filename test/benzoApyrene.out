 9 + 11 x + 3 x^2
 total: 23
